### ISSUE TITLE
Calendar outline

### ISSUE THREAD

**Issue by gamsim** [ez-connect/flutter-ionicons#2]
The calendar outline icon doesn’t render correctly.

**Comment by vnhdx** [comment]
Please try `v0.1.0` branch @gamsim 

```yml
dependencies:
  ionicons:
    git:
      url: https://github.com/ez-connect/flutter-ionicons.git
      ref: v0.1.0
```

**Comment by gamsim** [comment]
Thanks @thanh-vinh that looks good!

**Comment by vnhdx** [comment]
Fixed in v0.1.0
